gems/awesome-bot/CVE-2019-15224.yml

```diff
@@ -0,0 +1,21 @@
+---
+gem: awesome-bot
+url: https://nvd.nist.gov/vuln/detail/CVE-2019-15224
```

Comment:
Please use a different URL besides NVD for the `url`.

Reply:
Changed.